src/matrix.ts

```diff
@@ -16,8 +16,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import type Request from "request";
```

Comment:
I'm mildly concerned that this will cause webpack to bring in the whole request library when it shouldn't. A quick double check to see if the browser bundle is relatively sane (roughly the same size, as it doesn't exactly work right now) and that riot-web's webpack doesn't blow up would be appreciated.

Reply:
Before:
![image](https://user-images.githubusercontent.com/2403652/79964929-4ae00900-8483-11ea-9024-278413b81372.png)
After:
![image](https://user-images.githubusercontent.com/2403652/79965092-84187900-8483-11ea-9cbd-d9a4db80ad0f.png)

Somehow, its smaller 🤷 
![image](https://user-images.githubusercontent.com/2403652/79965222-a9a58280-8483-11ea-89f4-8dff7bd02e45.png)

and the build works so looks fine from my view